src/common.ts

```diff
@@ -1,11 +1,37 @@
-export interface TransactionOptions {
-  readOnly?: boolean;
-  timeout?: number;
-  exactStaleness?: number;
-  readTimestamp?: Date;
-  returnTimestamp?: boolean;
-  strong?: boolean;
+import {google as spTypes} from '../proto/spanner';
+import {Operation as GaxOperation} from 'google-gax/build/src/longrunning';
+import {ServiceError} from 'grpc';
+
+//tslint:disable-next-line no-any
+export type Any = any;
+export type BasicResponse = [IOperation];
```

Comment:
I think we could do a better job name this and `BasicCallback` here. `IOperation` isn't really that basic, and it looks like we already have a similar (more correct?) response below called `LongRunningOperationResponse`.

Reply:
I'll clean it all up asap